translator/GCP_cloud_function/cloud_function/main.py

```diff
@@ -42,8 +39,8 @@ def translate_newest_icone_to_wzdx(event, context):
   return
 
 def get_ftp_file(url) :
-  with closing(request.urlopen(url)) as r:
-    return r.read().decode('utf-8-sig')
+  return urllib.request.urlopen(url).read().decode('utf-8-sig')
+
 
 
 def parse_xml(xml_string):
```

Comment:
This `parse_xml` method is really just wrappinga single functionality - a call to the xmltodict library. Since we aren't doing anything else in here we aren't gaining anything by using this abstraction. Let's remove in favor of directly calling the `xmltodict.parse()` method.

Reply:
The intent of creating a separate function was to make refactoring/updating of the code easier. If in the future we decide to use a different xml parsing library or we want to add more parameters to the xmltodict.parse function call, then we only need to touch the parse_xml function, and we have unit tests that will ensure that the xml data is being parsed as expected